python/src/scipp/compat/mantid.py

```diff
@@ -853,6 +853,14 @@ def validate_dim_and_get_mantid_string(unit_dim):
         return known_units[user_k]
 
 
+def data_array_to_workspace_2d(data_array, dim, instrument_file=None):
+    return to_workspace_2d(x=data_array.coords[dim].values,
+                           y=data_array.values,
+                           e=data_array.variances,
```

Comment:
```suggestion
                           e=np.sqrt(data_array.variances),
```

Reply:
Doesn't `to_workspace_2d` take the sqrt?

Reply:
I had to double check but in this branch `mantid.py:894` takes the square root for us and we assert this in both that unit test and this

Good point to raise though, I had to double check myself